shell/platform/fuchsia/flutter/accessibility_bridge_unittest.cc

```diff
@@ -0,0 +1,297 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
May be we should also consider adding more test cases for sending partial updates.
Not sure if accessibility bridges supports it at this moment. But something to remember in the future.

Reply:
That's a good idea, but probably for a separate PR

Reply:
sounds good.